### PR TITLE
py-multiscale-run: v 0.5.1, 0.5

### PR DESCRIPTION
**Pull request by cattabiani**
py-multiscale-run: v 0.5.1, 0.5

update module

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py [hunk 1]
```diff
@@ -15,6 +15,8 @@ class PyMultiscaleRun(PythonPackage):
     maintainers("tristan0x", "cattabiani")
 
     version("develop", branch="main")
+    version("0.5.1", tag="0.5.1")
+    version("0.5", tag="0.5")
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py]: I'll allow it. But for future reference, we don't need intermediate versions :)

**Review by matz-e — DISMISSED** [review]

File: bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py (hunk 1 above)
Comment (cattabiani) [reply bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py]: tristan is annoyed that I created a 0.5.1 tag without a 0.5. The 0.5.1 is here. Anyway I move from here somebody complains

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py]: Yes, it should also be `0.5.0` not `0.5` :)

I'm done now :)

Comment (cattabiani) [reply bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py]: what if I just remove it? It has some problems anyway that version

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py]: Yes, remove faulty versions and towers.

**Review by matz-e — APPROVED** [review]

**Pull request merged**